src/IndexerInterface.php

```diff
@@ -44,7 +44,7 @@ interface IndexerInterface
 
     public const AVAILABLE_INDEXES = [
         ResultService::DELIVERY_RESULT_CLASS_URI => self::DELIVERY_RESULTS_INDEX,
-        TaoOntology::CLASS_URI_ASSEMBLED_DELIVERY => self::DELIVERIES_INDEX,
+        TaoOntology::CLASS_URI_ASSEMBLED_DELIVERY => self::DELIVERY_RESULTS_INDEX,
```

Comment:
may i ask you to explain why?

Reply:
It would be more coherent if we have something related to results, but not assembled delivery as key, but I know this was previously there.

Reply:
I rolled back this changes.